install/upgrade/versions/1.5.3.sh

```diff
@@ -19,4 +19,70 @@ upgrade_config_set_value 'UPGRADE_UPDATE_WEB_TEMPLATES' 'false'
 upgrade_config_set_value 'UPGRADE_UPDATE_DNS_TEMPLATES' 'false'
 upgrade_config_set_value 'UPGRADE_UPDATE_MAIL_TEMPLATES' 'false'
 upgrade_config_set_value 'UPGRADE_REBUILD_USERS' 'false'
-upgrade_config_set_value 'UPGRADE_UPDATE_FILEMANAGER_CONFIG' 'false'
\ No newline at end of file
+upgrade_config_set_value 'UPGRADE_UPDATE_FILEMANAGER_CONFIG' 'false'
+
+echo "[ * ] Updating apt keyring configuration..."
+
+if [ ! -f "/usr/share/keyrings/nginx-keyring.gpg" ]; then 
+    # Get Architecture
+    architecture="$(uname -m)"
+    case $architecture in 
+    x86_64)
+        ARCH="amd64"
+        ;;
+     aarch64)
+        ARCH="arm64"
+        ;;
+    *)
+        echo "Not supported"
+    esac
+        
+    #Get OS details
+    os=$(grep "^ID=" /etc/os-release | cut -f 2 -d '=')
+    codename="$(lsb_release -s -c)"
+    VERSION="$(lsb_release -s -r)"
+    mariadb_v=`mysql -V | awk 'NR==1{print $5}' | head -c 4`
+    
+    apt="/etc/apt/sources.list.d"
+    
+    if [ -f "$apt/nginx.list" ]; then
+        rm  $apt/nginx.list 
+        echo "   [ * ] NGINX"
+        echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/nginx-keyring.gpg] https://nginx.org/packages/mainline/$VERSION/ $codename nginx" > $apt/nginx.list
+        curl -s  https://nginx.org/keys/nginx_signing.key | gpg --dearmor | tee /usr/share/keyrings/nginx-keyring.gpg >/dev/null 2>&1
+    fi
+    if [ "$os" = "debian" ]; then
+        if [ -f "$apt/php.list" ]; then
+            rm  $apt/php.list 
+            echo "   [ * ] PHP"
+            echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/sury-keyring.gpg] https://packages.sury.org/php/ $codename main" > $apt/php.list
+            curl -s  https://packages.sury.org/php/apt.gpg | gpg --dearmor | tee /usr/share/keyrings/sury-keyring.gpg >/dev/null 2>&1
+        fi
+        if [ -f "$apt/apache2.list" ]; then
+            rm  $apt/apache2.list 
+            echo "   [ * ] Apache2"
+            echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/apache2-keyring.gpg] https://packages.sury.org/apache2/ $codename main" > $apt/apache2.list
+            curl -s https://packages.sury.org/apache2/apt.gpg | gpg --dearmor | tee /usr/share/keyrings/apache2-keyring.gpg >/dev/null 2>&1
+        fi
+    fi
+    if [ -f "$apt/mariadb.list" ]; then
+        rm  $apt/mariadb.list 
+        echo "   [ * ] MariaDB"
+        echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/mariadb-keyring.gpg] https://mirror.mva-n.net/mariadb/repo//$VERSION $codename main" > $apt/mariadb.list
+        curl -s https://mariadb.org/mariadb_release_signing_key.asc | gpg --dearmor | tee /usr/share/keyrings/mariadb-keyring.gpg >/dev/null 2>&1
```

Comment:
line 71 should be:

https ://mirror.mva-n.net/mariadb//repo/$mariadb_v/$VERSION $codename main